crates/matrix-sdk-ui/src/room_list_service/mod.rs

```diff
@@ -116,6 +116,7 @@ impl RoomListService {
         let mut builder = client
             .sliding_sync("room-list")
             .map_err(Error::SlidingSync)?
+            .share_pos()
```

Comment:
This is incorrect if `with_encryption` is true, because sharing the pos would require holding the cross-process crypto lock across the entire duration of a request/response cycle, and that's not happening (more code would need to be changed in the sliding sync itself).